src/components/Voting/Referendum/Items/ProposalItem/index.js

```diff
@@ -15,6 +23,30 @@ const endorseButton = (buttonEndorseCallback, userDidEndorse, referendumInfo) =>
     </Button>
   ));
 
+const blacklistButton = (proposalHash) => {
+  const dispatch = useDispatch();
+  const members = useSelector(congressSelectors.members);
+  const user = useSelector(blockchainSelectors.userWalletAddressSelector);
+
+  useEffect(() => {
+    dispatch(congressActions.getMembers.call());
+  }, [dispatch]);
+
+  const userIsMember = members.map((m) => m.toString()).includes(user);
```

Comment:
Leave a comment to refactor this after #143 is merged

Reply:
Addrd comment and created task on jira https://liberland.atlassian.net/jira/software/projects/BLOCKCHAIN/boards/2?selectedIssue=BLOCKCHAIN-222